add info about increase all ui size into FAQ for other new user to find it more easily (#30)
lite-xl/lite-xl.github.io: docs/about/faq.md

For example: `Core: Find Command` → `core:find-command`

> ## How to increase all UI size/scale?

Reviewer: ```suggestion
## How to make the Lite XL UI look larger?
```

> There are a few ways of setting the scale:

Reviewer: ```suggestion
Due to a limitation with the SDL2 library,
Lite XL is unable to get the scaling factor of your monitors.
There are a few ways to manually specify a scale:
```

> - If you use the settings plugin, you can find the scale options under Plugins -> Scale

Reviewer: Add a reference link to the settings plugin.

> - If you use the settings plugin, you can find the scale options under Plugins -> Scale

Reviewer: ```suggestion
- If you use the Settings GUI plugin, you can find the scale options under Plugins > Scale
```

> - If you want to set those via Lua, in your user module add

Reviewer: ```suggestion
- Otherwise, you can set them through Lua by adding the following lines to your user module:
```

>     ```lua

Reviewer: ```suggestion
    ```lua title="USERDIR/init.lua"
```

    config.plugins.scale.mode = "ui"

>     config.plugins.scale.default_scale = 2

Reviewer: check indent

    ```
    

> - Set the LITE_SCALE env var

Reviewer: ```suggestion
- Set the `LITE_SCALE` environment variable, e.g. running `LITE_SCALE=1.25 lite-xl` in your terminal.
```

> source,https://github.com/lite-xl/lite-xl/issues/2032#issuecomment-2711153517

Reviewer: ```suggestion
```

The docs are authoritative unless specified, no source needed here
